company-profiles/remotegarage.md

```diff
@@ -0,0 +1,33 @@
+
+# Remote Garage
+
+## Company blurb
+
+[Remote Garage](http://www.remotegarage.club/) is on a mission to bring your best potential regardless of what left undiscovered by becoming the first school in India to launch a new career. Sound like a worthy endeavor? Come join us!
+
+We are a community of gophers, who do programming not because they expect to get paid or get adulation by public, But because it is fun to program.
+
+
+## Company size
+
+10+ employees (as of Aug 2020).
+
+## Remote status
+
+We highly believe and focus on providing remote employement.
```

Comment:
Bringing Traditional Education to the way it was. Education should be free.	

Remote Garage works upon the concept of gurudakshna which allows anyone to learn the most demanding Technologies and prepares you to work with the opensource community across the globe with zero Investment.